tests/test_repro.py

```diff
@@ -1378,3 +1368,58 @@ def test(self):
 
         expected_metrics_display = "{}: {}".format(metrics_file, metrics_value)
         self.assertIn(expected_metrics_display, self._caplog.text)
+
+
+# Pytest style tests
```

Comment:
We don't mark them like so in other places. Let's remove this comment as well.

Reply:
Or we can mark them like this everywhere. Header comments do help with infile navigation when it contains several logical sections.

Reply:
I don't think it provides enough value. It is not like unittests are hard to distinguish from pytest tests. Either way, we should either mark them everywhere like this, or not mark at all and leave it for later, since it is not the scope of this issue.